### ISSUE TITLE
Trovebox as a Labs project

### ISSUE THREAD

**Issue by rgrp** [okfn/okfn.github.com#426]
There is discussion about having Trovebox open photo project live at labs with Open Knowledge (Intl) as the holder and curator of the assets:

https://github.com/photo/frontend/issues/1570

Question: should Labs take this on?

/cc @pwalsh @danfowler @davbre @davidmiller (and all labs members)

### How this would work

*This is a rough sketch - for discussion and review*

* Trovebox (photo) github org would stay where it was but relevant people (e.g. OK sysadmins) would become owners
* Other trovebox assets (what are they?) might move - fairly limited (e.g. domain name)
* Copyright assignment (?)
* A contact person for the project would be established e.g. those currently offering to maintain it

### What would be required from Labs

* There is an offer of a maintainer from the community who could become maintainer of this within Labs
* Current Labs members would *not* be expected to take on maintaining the software
* There is no plan to host a high maintenance service or similar - only to hold the software assets

### Why

Why is trovebox aligned with Labs mission and vision?

* Trovebox is about supplying open services that allow people to manage their (photo) content. Labs wants to support infrastructure that allows people to publish open material or manage their own material in an open, non-proprietary way.
* Excellent, high quality product that has been orphaned but where the is community interest in maintaining and developing it

**Comment by tfmorris** [comment]
I would have thought that IP transfers would require involvement of an officer of the OKFN corporation, rather than a popular vote of labs volunteers.

If they decide not to take it on, the [Software Freedom Conservancy](https://sfconservancy.org/supporter/) is an organization which will hold funds, assets like domain names, etc for approved open source organizations.  The Apache Foundation might be another alternative, although it would be a bit more of a stretch. Neither is probably interested in taking on an abandoned project and it isn't clear to me just how active the TroveBox community is. At least on Github, it doesn't appear to be very active as measured by commits or activity on forks.

**Comment by rgrp** [comment]
@tfmorris 

> I would have thought that IP transfers would require involvement of an officer of the OKFN corporation, rather than a popular vote of labs volunteers.

Yes, of course, any formal transfer would require that but it would be predicated on and follow a discussion like this in Labs.

Small point: it is always Open Knoweldge or Open Knowledge X rather than OKFN ...

On the other point the discussion is currently with Labs as we have talked with them in the past and there is an alignment. Understand there are other options but let's focus discussion here on PoV of Labs as that is the matter at hand :smile:

**Comment by danfowler** [comment]
@rgrp I'm not strongly feeling the connection between Trovebox and Labs.  I get that Labs and OK sysadmins might not directly have to support this work, but there is still some cost to running and putting our name on a service that is mostly abandoned.  Can adopting Trovebox be strategically important in further developing other Labs projects?

**Comment by rgrp** [comment]
@danfowler good questions. I think we should also ping okfn-labs on this one with a link here and see if people have any thoughts.

**Comment by jmathai** [comment]
Thanks for the discussion on this. Here's some additional input.

The Trovebox project has been largely dormant for about 1.5 years since the core team moved on. One of the community members wants to take the lead on continuing its development.

As part of the core team moving on the CLAs that were held were transferred to a company that sought to integrate Trovebox into their suite of products. Not all contributors signed a CLA.

We renamed the community project "The Photo Project" due to the company mentioned above and another trademark issue we had with the name OpenPhoto. 

@danfowler your point is valid and I want to make sure that it would make sense to do this.

**Comment by rgrp** [comment]
@jmathai to clarify on the code "ownership": i assume that everything currently in the photo org is openly licensed? i.e. the code that was open before the core team moved on remained open?

**Comment by jmathai** [comment]
@rgrp Everything we had was licensed under Apache2. So to the best of my understanding it remains under that license. There was an asset purchase agreement where IP assigned to our company was transferred. This included IP from the founders, employees and contractors who signed copyright assignment agreements.

About 1/2 of the volunteer contributors signed a copyright license agreement.

Hope that answers your question.

**Comment by rgrp** [comment]
@jmathai that's pretty clear and really useful -- basically everything in the github repo is licensed under Apache2 though the underlying copyright went with the copyright assignment. The existing code can therefore be used under Apache2 and new code would obviously be copyright the contributors.

**Comment by jmathai** [comment]
> though the underlying copyright went with the copyright assignment.

Correct for work which was compensated. Copyright for all non-paid work remains with the authors.
